classes/class3/README.md

```diff
@@ -1 +1,364 @@
+```
+hello world
+```
+Awesome! You just ran your first application using Express!
+
+Express makes writing web servers in Node much easier. Here are some of the important differences between the examples you did in the book and Express:
+* Routing. Instead of trying to parse the URL the user is at ourselves, we can just tell Express to match an individual path with a function.
+* Sending a response. Express takes care of setting many obvious response headers for you. Express will also handle sending files like images, music, audio, or `.html` files from a folder easily.
+* Handling templates. We haven't gotten here yet, but we'll touch on this later.
+Let's go back to our `app.js`. We want more than just hello world. Let's make Express show the string `hello olin` when we go to `http://localhost:3000/olin`.
+
+Below the `app.get('/',...` statement, add the following:
+```javascript
+app.get('/olin', function(req, res){
+  res.send('hello olin');
+});
+```
+
+So what does `app.get` do? It tells express that every time we have a `GET` request from a client (the browser). We'll learn more about this in Routing.
+###Routing
+Routing is the process of serving up different pages for different urls. When you go to www.mycoolsite.com/ your computer goes out on the internet and asks mycoolsite's server for a page. Mycoolsite's server then sees that request and sends back information to your computer in the form of html. This html is then rendered on your browser.
+
+If you go to www.mycoolsite.com/ mycoolsite's servers obviously can't send you the same data it sent www.mycoolsite.com/olin. So mycoolsite's servers needs to differentiate `/` from `/olin`. This process is known as routing.
+
+In the Node Beginner Book, we did routing through something like
+```javascript
+var pathname = url.parse(request.url).pathname;
+route(handle, pathname, response, request);
+```
+Instead of writing the code to handle the route ourselves, Express does it for us with these statements:
+```javascript
+app.get('/', function (req, res){
+  res.send('hello world');
+});
+
+app.get('/olin', function (req, res){
+  res.send('hello olin');
+});
+```
+These two routes we created are for the index page (/) and the olin page (/olin).
+
+So what does `app.get` do? It tells express that every time that particular route (the first string argument) receives a `HTTP GET` request, we want to execute the anonymous function (the second argument). `HTTP` lets you perform different types of requests for a particular route, and these types are called **methods**. `GET` is just one of the methods you can perform, and is the most common (every time request an image or a script for example). You can [view the other types of HTTP methods on wikipedia](http://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods). But for now, let's only consider these two:
```

Comment:
We already talked about this above... Just remove that whole section, since it is being repeated in its entirety + more, here.